micronaut-graalpy/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id "io.micronaut.build.internal.graal-languages-module"
+}
+
+// XXX GP
+// Execution failed for task ':micronaut-graalpy:findBaseline'.
+// > Could not find a previous version for 1.0.0
+findBaseline.enabled = false
+
+repositories {
+    // XXX GP
+    maven {
+        url "file:///tmp/local_repo"
+    }
+    mavenCentral()
+}
+
+dependencies {
+    // XXX GP how to declare graal version?
+    // XXX GP what python edition? -> depend on python, exclude truffle enterprise exclude group: "org.graalvm.truffle", module: "truffle-enterpise"
+    //implementation("org.graalvm.python:python:24.2.0@pom") {
+    implementation("org.graalvm.python:python:24.2.0@pom") {
+        transitive = true
```

Comment:
transitve = false is not enough, you have to add individual excludes

Reply:
i'm confused, it says `transitive = true` 
it is a pom dependency and if not set, then in the generated `pom.xml` everything is excluded